app/views/podcast/SinglePodcastView.vue

```diff
@@ -67,6 +69,25 @@ export default {
     await this.fetchPodcast({ podcastId: handle  })
     this.podcast = this.getPodcast(handle)
     this.loaded = true
+  },
+  metaInfo () {
+    // instead of using normal static template, we use podcast.static.pug where these values are not present
+    const podcast = this.podcast
+    const title = `${i18n(podcast, 'name')} with ${i18n(podcast, 'guestName')}`
+    const image = `${window.location.protocol}//${window.location.host}/images/pages/podcast/edtech-adventure-og-image.jpg`
+    const desc = `${podcast?.description ? marked(i18n(podcast, 'description')).replace(/<[^>]*>?/gm, '') : ''}`
```

Comment:
do we need to make sure the marked won't generate any XSS attack ?

Reply:
Ahh, good observation. However, it should be ok here since the content will be added by our team only!